apps/aens/src/aens_hash.erl

```diff
@@ -28,14 +34,38 @@
 %%% API
 %%%===================================================================
 
+
 -spec commitment_hash(binary(), integer()) -> commitment_hash().
 commitment_hash(NameAscii, Salt) ->
-    NameHash = name_hash(NameAscii),
-    SaltBin = int_to_bin(Salt),
-    hash(<<NameHash/binary, SaltBin/binary>>).
+    case aens_utils:name_domain(NameAscii) of
+        {ok, Domain} when Domain =:= <<"aet">> ->
```

Comment:
Shouldn't it be related to `aec_governance` somehow?

Reply:
Yes, indeed.